plugin/pkg/parse/host.go

```diff
@@ -63,6 +63,12 @@ func HostPortOrFile(s ...string) ([]string, error) {
 		}
 
 		if net.ParseIP(stripZone(addr)) == nil {
+			if strings.HasPrefix(host, "unix://") {
```

Comment:
We could include the unix schema in [plugin/pkg/parse/transport.go](https://github.com/coredns/coredns/blob/master/plugin/pkg/parse/transport.go)